helm-chart/values.yaml

```diff
@@ -92,3 +96,21 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+volumes: []
+# volumes:
```

Comment:
you don't need this line

Reply:
it may help someone who consumes the helm to see what the options are there without getting into the chart specs

Reply:
What I mean is that `# volumes:` is not really needed, especially if line above defines it.
I'm not going to be mental about it though :)

Reply:
True, I thought comment with a sample can guide someone who sees it the 1st time:)